pkg/scheduler/core/generic_scheduler.go

```diff
@@ -211,12 +211,15 @@ func (g *genericScheduler) selectHost(nodeScoreList framework.NodeScoreList) (st
 
 // numFeasibleNodesToFind returns the number of feasible nodes that once found, the scheduler stops
 // its search for more feasible nodes.
-func (g *genericScheduler) numFeasibleNodesToFind(numAllNodes int32) (numNodes int32) {
-	if numAllNodes < minFeasibleNodesToFind || g.percentageOfNodesToScore >= 100 {
+func (g *genericScheduler) numFeasibleNodesToFind(prof *profile.Profile, numAllNodes int32) (numNodes int32) {
```

Comment:
Prefer to expose it as `numFeasibleNodesToFind(numAllNodes, profPercentageOfNodesToScore int32)`